packages/relay/tests/lib/eth/eth_getLogs.spec.ts

```diff
@@ -281,6 +281,45 @@ describe('@ethGetLogs using MirrorNode', async function () {
     expectLogData3(result[2]);
   });
 
+  [CONTRACT_ADDRESS_1, [CONTRACT_ADDRESS_1]].forEach((address) => {
+    it(`single address filter \`${JSON.stringify(address)}\` and large block range`, async function () {
```

Comment:
not a fan of using a dynamic name for the test case, I understand the intent but I think is the same effect if the loop is inside the test. That would help readability and maintenability. And in the case the test fail on the CI, to find it faster on the codebase

I could suggest the name: `should filter logs by single address or array of addresses with a large block range` instead?

Reply:
As discussed with @AlfredoG87, we left the dynamic test cases, but improve the test title to match `it should` format.